python/test/test_comb.py

```diff
@@ -110,7 +120,10 @@ def test_make_approximate():
         dp.c.make_fix_delta(dp.c.make_zCDP_to_approxDP(dp.m.make_gaussian(*input_space, 10.)), delta=1e-6)
     ])
 
-    print(meas.map(1.))
+    assert meas.map(1.) == (0.5299414688369495, 1e-06)
```

Comment:
Not an absolute blocker, but prefer an expression that explains where this number comes from. I imagine a future maintainer who is hitting a test failure here: They shouldn't "fix" it by just changing the assertion to whatever number they have, and we should set a good example.